lgtm.yml

```diff
@@ -0,0 +1,9 @@
+queries:
+- exclude: py/insecure-default-protocol
+  # For Python>=3.6, ssl.SSLContext by default uses secure settings.
+  # See https://docs.python.org/3.6/library/ssl.html#ssl.SSLContext
```

Comment:
This is a workaround to disable LGTM.com _deprecated_ checking of SSL insecure default protocol.

Sadly, this will only become effective after merging.